### PR TITLE
Simplify INT-MX and Telemetry Report Processing

### PR DESCRIPTION
**Pull request by mickeyspiegel**
In INT spec:
- Delete INT-MX padding option, relying only on clearing of
  instruction bits
- Delete E, M, and Hop ML fields from the INT-MX header format
- INT-MX 'source-only' processing simplification, no longer
  describing the option using domain specific extension data
- Modify description of INT-MX local metadata processing so that
  it is not transit node specific, instead applies to source,
  transit, and sink
- INT-MD local metadata processing applies to source node as well
  as transit
- Add processing of local metadata at INT-MD sink, with two options
  (these options are already mentioned in the Telemetry Report spec,
  but without details, which now exist in the INT spec)
  1. Add to metadata stack, pointing to INT-MD transit node processing
  2. Add to Telemetry Report's "Variable Optional Baseline Metadata"
     and "Variable Optional Domain Specific Metadata", pointing to
     INT-MX local metadata processing with one addition and one
     restriction.
- Minor editorial cleanup
  - "INT-Source" --> "INT Source"
  - Fix incorrect hardcoded reference
  - "section" --> "Section" for consistency
  - "DSMdbits" --> "DSMdBits"

In Telemetry Report spec:
- Delete padding option, relying only on clearing of bits for
  unavailable metadata or reserved bits

### CODE REVIEW

File: telemetry/specs/INT.mdk [hunk 1]
```diff
@@ -1288,11 +1288,37 @@ INT-MD Metadata Header and Metadata Stack:
...
+  - If an INT hop does not add metadata to a packet due to any of the
     above reasons, it must not decrement the *Remaining Hop Count* in the INT
     metadata header.
+
+* The INT Sink node has the option to add its local telemetry metadata in either
+  of the following ways, with differing implementation dependent impact:
+
+  1. The INT Sink's local telemetry metadata may be added to the INT-MD metadata
+     stack by following the same procedures described just above for INT
+     Transit nodes. The metadata stack with the INT Sink's local telemetry
```
Comment (jklr) [inline telemetry/specs/INT.mdk:1302]: Transit nodes => Source/Transit nodes 
?

File: telemetry/specs/telemetry_report.mdk [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Title : Telemetry Report Format Specification
 Title Note : Version 2.0
-Title Footer : 2018-05-27
+Title Footer : 2018-06-11
```
Comment (jklr) [inline telemetry/specs/telemetry_report.mdk]: 2020-06-11?

**Review by jklr — APPROVED** [review]
Thanks for the revision. 
Can we change the MX source-only example to use a separate UDP-INT header? That encap can cover wider usecase.

File: telemetry/specs/telemetry_report.mdk (hunk 2 above)
Comment (mickeyspiegel) [reply telemetry/specs/telemetry_report.mdk]: fixed

**Comment by jklr** [comment]
The example LGTM. Thx.

**Comment by jklr** [comment]
shall we merge?

**Comment by jklr** [comment]
@rsivakolundu this is the PR. pls review.

**Comment by rsivakolundu** [comment]
Will review it tonight.

Sorry for the delay.

-Ramesh

**Review by rsivakolundu — APPROVED** [review]
Looks good.

**Pull request merged**